globsim/interpolate/GenericInterpolate.py

```diff
@@ -13,6 +14,14 @@
 from globsim.boundingbox import stations_bbox, netcdf_bbox, BoundingBox
 
 logger = logging.getLogger('globsim.interpolate')
+logger.setLevel(logging.DEBUG)
+fh = logging.FileHandler('globsim_interpolate.log')
+fh.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+fh.setFormatter(formatter)
+
+logger.addHandler(fh)
```

Comment:
don't need to put this here.

* You can [set the logging level](https://github.com/geocryology/globsim/blob/9562abfea51498bd8253a4ad8e5fa58735b2fa69/globsim/globsim_cli.py#L68) when calling the command line function with the `-v` flag

e.g. `globsim interpolate -v` or `globsim interpolate -v 10` (for `logging.DEBUG`)


* You can [specify a log file](https://github.com/geocryology/globsim/blob/9562abfea51498bd8253a4ad8e5fa58735b2fa69/globsim/globsim_cli.py#L70) when calling the command line function with the `-L` flag

e.g. `globsim interpolate -L log.txt`